### DISCUSSION TITLE
06/06/22 Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [AngellXR/metaversedev#16] [General]
Agenda: 
- [selecting a license](https://github.com/AngellXR/metaversedev/issues/6)
- Discussing our early [curriculum](https://github.com/AngellXR/metaversedev/discussions/17)

📺 [Recording](https://drive.google.com/file/d/15dowLmP4vs1P5hX0ROfb3U_oH4yApcEC/view?usp=drivesdk)

**Comment by mrmetaverse** [comment]
https://github.com/AngellXR/metaversedev/milestone/3

**Reply by msub2** [reply]
We should probably all talk through the actual curriculum before working out a potential website design, yeah?

**Reply by mrmetaverse** [reply]
Yep, absolutely. So in order to even consider v 1, we must have an idea of an early curriculum. I'll add that to the milestone.

**Comment by mrmetaverse** [comment]
We like the idea of something open and accessible and content rich. Easy to contribute to, or possessing a well-sourced pool of experts to draw knowledge from if we are to create our own curriculum.

**Comment by mrmetaverse** [comment]
Michael Sim shared this: https://www.pluralsight.com/ they used to work there

**Comment by mrmetaverse** [comment]
If someone wants to get into the Metaverse as a creator and have no idea how to, tools, tooling, interop, etc. I want to be able to advice them as I am in touch with a lot of others. I want to help guide newcomers in the right direction. Something open, where people can contribute to.

**Comment by mrmetaverse** [comment]
RE: License #6 

In interest of being open and accessible, we feel MIT is the best start. We may want apache in the future but MIT feels more than fine for now. 

>№4: MIT License
>Next up is many developers’ favorite license — including myself —[ the MIT license](https://opensource.org/licenses/MIT). The MIT license is a permissive license originally released by the Massachusetts Institute of Technology in the late ’80s. The reason many prefer this license is that it’s short, simply written, and clear about what is allowed and what’s not.
>The MIT license is one of the most used licenses in the open-source world today. Basically, this license allows you to do whatever you want to the licensed material as long as the original copyright and license are included in your work files. It removes any liability from authors and does not explicitly contain a patent grant.

**Comment by mrmetaverse** [comment]
Yesterday's metaverse.dev team sync was recorded. You may access [here](https://drive.google.com/file/d/15dowLmP4vs1P5hX0ROfb3U_oH4yApcEC/view?usp=drivesdk) 📺

**Comment by mrmetaverse** [comment]
# Homework: 

https://github.com/AngellXR/metaversedev/discussions/17#discussioncomment-2893576
